rpmbuild/copr-rpmbuild.spec

```diff
@@ -104,7 +106,9 @@ Requires: nosync
 %endif
 Requires: openssh-clients
 Requires: podman
-%if !0%{?openEuler}
+%if 0%{?openEuler} > 0 || 0%{?rhel} > 0
+# not supported
+%else
```

Comment:
This means that we make copr-rpmbuild unusable on RHEL at all? (=not just emulated builds).

Reply:
ok, I see thre are two commits